s/elements/bindings-editor/utils/prepare-assign-keybind.ts

```diff
@@ -0,0 +1,27 @@
+
+import {AssignKeybind} from "../types.js"
+import {Bindings} from "../../../types.js"
+import {NubContext} from "../../context/element.js"
+
+export function prepareAssignKeybind(context: NubContext): AssignKeybind {
```

Comment:
the purpose of this function was to avoid `context` from trickling all the way down to the panel views